kraken-std/src/kraken/std/python/tasks/pytest_task.py

```diff
@@ -42,6 +56,13 @@ def get_execute_command(self) -> list[str] | TaskStatus:
         ]
         command += flatten(["--ignore", str(self.project.directory / path)] for path in self.ignore_dirs.get())
         command += ["--log-cli-level", "INFO"]
+        if self.coverage.is_filled():
+            coverage_file = f"coverage{self.coverage.get().get_suffix()}"
+            command += [
+                "--cov-report",
+                f"{self.coverage.get().get_format()}:{str(self.project.build_directory / coverage_file)}",
+                f"--cov={str(self.project.directory / self.settings.source_directory)}",
+            ]
```

Comment:
Should there be a check to see if `pytest-cov` is installed before adding these parameters?